modules/client-common/src/main/java/org/apache/ignite/internal/jdbc/proto/event/JdbcColumnMeta.java

```diff
@@ -388,14 +394,20 @@ private static String typeName(String cls) {
             return "VARCHAR";
         } else if (byte[].class.getName().equals(cls)) {
             return "BINARY";
-        } else if (Time.class.getName().equals(cls)) {
+        } else if (Time.class.getName().equals(cls) || LocalTime.class.getName().equals(cls)) {
             return "TIME";
-        } else if (Timestamp.class.getName().equals(cls)) {
+        } else if (LocalDateTime.class.getName().equals(cls)) {
+            return "DATETIME";
+        } else if (Timestamp.class.getName().equals(cls) || Instant.class.getName().equals(cls)) {
             return "TIMESTAMP";
-        } else if (Date.class.getName().equals(cls) || java.sql.Date.class.getName().equals(cls)) {
+        } else if (Date.class.getName().equals(cls) || java.sql.Date.class.getName().equals(cls) || LocalDate.class.getName().equals(cls)) {
             return "DATE";
         } else if (BigDecimal.class.getName().equals(cls)) {
             return "DECIMAL";
+        } else if (Duration.class.getName().equals(cls)) {
+            return "DURATION";
```

Comment:
neither DURATION nor PERIOD are database types

Reply:
I don`t know strict definition of database types, if we follow java.sql.Types than we need to define smt like JAVA_OBJECT here and in string representation.